wxRuby3 1.0.0 release post (#94)
wxWidgets/website: _posts/2024-05-18-wxruby3-1.0.0-released.md

---
title: "wxRuby3 1.0.0 released"
date: 2024-05-18
comments: true
---

> It's time! wxRuby3 has reached it's 1.0.0 stable release.

Reviewer: ```suggestion
It's time! wxRuby3 has reached its 1.0.0 stable release.
```

This release targets the wxWidgets 3.2.5 release by default, which contains (among many others) quite some fixes which 
came from the development of wxRuby3, and can be installed from [RubyGems](https://rubygems.org/gems/wxruby3) or 
downloaded from [Github](https://github.com/mcorino/wxRuby3/releases/download/v1.0.0/wxruby3-1.0.0.gem). 

This release is the result of about 1.5 years of development, starting at the abandoned remains of wxRuby2, taking a 
page out of the wxPython Phoenix book and ending with this release.

wxRuby3 is a cross-platform GUI library for Ruby, based on the mature wxWidgets GUI toolkit for C++. It uses native 

> widgets wherever possible, providing the correct look, feel and behavior to GUI applications on Windows, OS X and 

Reviewer: ```suggestion
widgets wherever possible, providing the correct look, feel and behavior to GUI applications on Windows, macOS and 
```

Linux/GTK. wxRuby aims to provide a comprehensive solution to developing professional-standard desktop applications in 
Ruby.

> wxRuby3 also provides extensive usage documentation at it's [Wiki](https://github.com/mcorino/wxRuby3/wiki) as well as 

Reviewer: ```suggestion
wxRuby3 also provides extensive usage documentation at its [Wiki](https://github.com/mcorino/wxRuby3/wiki) as well as 
```

a complete [API reference](https://mcorino.github.io/wxRuby3/index.html). 

## What has changed in this release?

* improved documentation

> * support building against wxWidgets HEAD version

Reviewer: ``
```suggestion
* support building against wxWidgets GIT master version
```
